packages/application-components/src/components/dialogs/form-dialog/form-dialog.spec.tsx

```diff
@@ -1,104 +1,95 @@
-import { Dispatch, useState } from 'react';
-import SecondaryButton from '@commercetools-uikit/secondary-button';
-import { screen, renderComponent, fireEvent } from '../../../test-utils';
+import { screen, fireEvent } from '../../../test-utils';
+import {
+  DialogCustomTitle,
+  TDialogComponentValidator,
+  createDialogValidator,
+} from '../internals/dialog-test-utils';
 import FormDialog from './form-dialog';
 
-type DialogControllerProps = {
-  children: (renderProps: {
-    isOpen: boolean;
-    setIsOpen: Dispatch<boolean>;
-  }) => JSX.Element;
-};
-const DialogController = (props: DialogControllerProps) => {
-  const [isOpen, setIsOpen] = useState(false);
-  return (
-    <div>
-      <SecondaryButton
-        label="Open Form Dialog"
-        onClick={() => setIsOpen(true)}
-      />
-      {props.children({ isOpen, setIsOpen })}
-    </div>
-  );
+const createFormDialogValidator = () => {
+  const onCancel = jest.fn();
+  const onConfirm = jest.fn();
+
+  return createDialogValidator({
+    component: FormDialog,
```

Comment:
I would appreciate some help with this TS error as I don't seem to be able to find the right configuration.

Reply:
Try typing the prop as `ElementType`

Reply:
Updated here: https://github.com/commercetools/merchant-center-application-kit/commit/c31bc90c72b996181b5261a2fd4a9aee3d860509